vignettes/advs.Rmd

```diff
@@ -79,28 +73,33 @@ metadata for this package are as follows:
 
 ### Reference Files for Parameters by Age
 
-As growth rates and patterns differ between sex, different reference
-values are used accordingly and for different age groups.
-
-For the applicable parameters here (height/weight/BMI), we made the
-default decision to rely on the WHO standards for children \<2 years of
-age (\<730.5 days) and CDC standards for children \>=2 years of age
-(\>=730.5 days). This is made in accordance with the [CDC
-recommendation](https://www.cdc.gov/growthcharts/who_charts.htm).
-
-So, the first step here needed is to combine the various metadata by
-parameter and ensuring sex and age are shown in a consistent way.
-For weight-based indicators we keep WHO and CDC metadata separate due to the WHO
-adjustment (restricted application of the LMS method) needed for the
-anthropometric indicator derivations.
-
-Additionally, as the CDC growth charts only offer monthly intervals we
-took advice from an ex-CDC expert in this area and made the decision in
-our template to add interpolation code so that we have a record for each
-day of age. When converting the CDC months into days we use rounding
-to bring them to a whole number of days. If you would prefer to rely
-solely on the monthly chart here, then you could remove the rounding
-and the call to the `derive_interp_records()` function.
+As growth rates and patterns by age and sex, reference values differ by these
+two characteristics. 
```

Comment:
@Fanny-Gautier @zdz2101 does this sentence read well to you? it doesn't make sense to me. i found the old wording clearer but might just be me

Reply:
I am not a native English speaker, I had first to translate it to ensure I properly understood it... We can revert it or change it again if you prefer. I am totally fine with this option ;-)

Reply:
Revert it please. I think its missing a word or 2 for it to read properly.

Reply:
I think that the correct sentence should have been "As growth rates and patterns **differ** by age and sex, reference values differ by these two characteristics. "

What do you think instead of "As growth rates and patterns differ between sex, different reference values are used accordingly and for different age groups.", as the part "and for different age groups." is also a bit weird.